katello_certs_tools/katello_ssl_tool.py

```diff
@@ -164,20 +184,23 @@ def genPrivateCaKey(password, d, verbosity=0, forceYN=0):
     finally:
         chdir(cwd)
 
-    out = out_stream.read(); out_stream.close()
-    err = err_stream.read(); err_stream.close()
+    out = out_stream.read()
+    out_stream.close()
+    err = err_stream.read()
+    err_stream.close()
+
     if ret:
         raise GenPrivateCaKeyException("Certificate Authority private SSL "
                                        "key generation failed:\n%s\n%s"
                                        % (out, err))
     if verbosity > 2:
         if out:
-            print "STDOUT:", out
+            print("STDOUT:", out)
         if err:
-            print "STDERR:", err
+            print("STDERR:", err)
 
     # permissions:
-    os.chmod(ca_key, 0600)
+    os.chmod(ca_key, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)